lan867x/examples/lan867x_tcp_sockets/server/main/server.c

```diff
@@ -0,0 +1,103 @@
+#include <stdio.h>
+#include <string.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_netif.h"
+#include "esp_netif_net_stack.h"
+#include "esp_eth.h"
+#include "esp_event.h"
+#include "esp_log.h"
+#include "ethernet_init.h"
+#include "lwip/sockets.h"
+
+#define SOCKET_PORT         5000
+#define LISTENER_MAX_QUEUE  10
```

Comment:
Nitpick, you can decrease the listener queue to minimum, as you're serving only one client

Reply:
Done.